Formula/corsixth.rb

```diff
@@ -16,64 +15,30 @@ class Corsixth < Formula
 
   depends_on "cmake" => :build
   depends_on "luarocks" => :build
-  depends_on xcode: :build
   depends_on "ffmpeg"
   depends_on "freetype"
-
-  # This PR implements a limited form of lua 5.4 support:
-  # https://github.com/CorsixTH/CorsixTH/pull/1686
-  # It breaks some features.  Maintainer does not appear to have intentions of
-  # supporting lua 5.4.
-  depends_on "lua@5.3"
-
+  depends_on "lua"
   depends_on "sdl2"
   depends_on "sdl2_mixer"
 
-  resource "lpeg" do
-    url "http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.2.tar.gz"
-    mirror "https://sources.voidlinux.org/lua-lpeg-1.0.2/lpeg-1.0.2.tar.gz"
-    sha256 "48d66576051b6c78388faad09b70493093264588fcd0f258ddaab1cdd4a15ffe"
-  end
-
-  resource "luafilesystem" do
-    url "https://github.com/keplerproject/luafilesystem/archive/v1_7_0_2.tar.gz"
-    sha256 "23b4883aeb4fb90b2d0f338659f33a631f9df7a7e67c54115775a77d4ac3cc59"
-  end
-
   def install
-    # Make sure I point to the right version!
-    lua = Formula["lua@5.3"]
-
-    ENV["TARGET_BUILD_DIR"] = "."
-    ENV["FULL_PRODUCT_NAME"] = "CorsixTH.app"
+    lua = Formula["lua"]
 
-    luapath = libexec/"vendor"
-    ENV["LUA_PATH"] = "#{luapath}/share/lua/5.3/?.lua"
-    ENV["LUA_CPATH"] = "#{luapath}/lib/lua/5.3/?.so"
+    args = std_cmake_args + %W[
+      -DLUA_INCLUDE_DIR=#{lua.opt_include}/lua
+      -DLUA_LIBRARY=#{lua.opt_lib}/liblua.dylib
+      -DCORSIX_TH_DATADIR=#{prefix}/CorsixTH.app/Contents/Resources/
+      -DWITH_LUAROCKS=ON
```

Comment:
I don't think we want this -- the resources makes it clearer that the extra downloads are versioned and checksummed.

Reply:
It also looks like this option makes `corsixth` vendor its own Lua, which we don't want.

Reply:
tackle on it now.